src/qst.cpp

```diff
@@ -14484,23 +14382,7 @@ int32_t readguys(PACKFILE *f, zquestheader *Header)
 		for ( int32_t q = 0; q < 8; q++ ) tempguy.initD[q] = 0; //Script Data
 		for ( int32_t q = 0; q < 2; q++ ) tempguy.initA[q] = 0; //Script Data
 		
-		tempguy.misc16 = 0;
-		tempguy.misc17 = 0;
-		tempguy.misc18 = 0;
-		tempguy.misc19 = 0;
-		tempguy.misc20 = 0;
-		tempguy.misc21 = 0;
-		tempguy.misc22 = 0;
-		tempguy.misc23 = 0;
-		tempguy.misc24 = 0;
-		tempguy.misc25 = 0;
-		tempguy.misc26 = 0;
-		tempguy.misc27 = 0;
-		tempguy.misc28 = 0;
-		tempguy.misc29 = 0;
-		tempguy.misc30 = 0;
-		tempguy.misc31 = 0;
-		tempguy.misc32 = 0;
+		for ( int32_t q = 0; q < 32; q++) tempguy.attributes[15+q] = 0; //misc 16-32
```

Comment:
Out of bounds on the array here
```suggestion
		for ( int32_t q = 15; q < 32; ++q) tempguy.attributes[q] = 0; //misc 16-32
```